lib/workers/mixins/web_server_worker_mixin.rb

```diff
@@ -10,8 +10,6 @@ def do_work
 
   def do_before_work_loop
     @worker.release_db_connection
-
-    # Since thin traps interrupts, log that we're going away and update our worker row
```

Comment:
@imtayadeway Do you know if puma also traps SIGINT/SIGTERM?

If so, I'd rather we drop the at_exit logging in a followup PR in favor of rescuing SignalException, see [here](https://github.com/ManageIQ/manageiq/pull/2386/files#diff-4ef749ed3132f4a0d2c09badafae1034R124)